tests/modules/nf-core/rtgtools/rocplot/main.nf

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { RTGTOOLS_ROCPLOT } from '../../../../../modules/nf-core/rtgtools/rocplot/main.nf'
+include { RTGTOOLS_VCFEVAL } from '../../../../../modules/nf-core/rtgtools/vcfeval/main.nf'
+include { UNTAR            } from '../../../../../modules/nf-core/untar/main.nf'
+include { HAPPY_HAPPY      } from '../../../../../modules/nf-core/happy/happy/main.nf'
+
+workflow test_rtgtools_rocplot {
+
+    input = [
+        [ id:'test' ], // meta map
+        file(params.test_data['homo_sapiens']['illumina']['test2_haplotc_vcf_gz'], checkIfExists: true),
+        file(params.test_data['homo_sapiens']['illumina']['test2_haplotc_vcf_gz_tbi'], checkIfExists: true),
+        file(params.test_data['homo_sapiens']['illumina']['test2_haplotc_ann_vcf_gz'], checkIfExists: true),
+        file(params.test_data['homo_sapiens']['illumina']['test2_haplotc_ann_vcf_gz_tbi'], checkIfExists: true),
+        file(params.test_data['homo_sapiens']['genome']['genome_21_multi_interval_bed'], checkIfExists: true),
+        file(params.test_data['homo_sapiens']['genome']['genome_bed'], checkIfExists: true)
+    ]
+
+    compressed_sdf = [
+        [ id:'test' ],
+        file(params.test_data['homo_sapiens']['genome']['genome_21_sdf'])
+    ]
+
+    sdf = UNTAR( compressed_sdf ).untar
+
+    RTGTOOLS_VCFEVAL ( input, sdf )
```

Comment:
Shouldn't there be an output of this process in the test data repo so it doesn't have to keep regenerating in CI? Or is it really trivial?

Reply:
It's a very low footprint process, but I can add it if you would prefer it :)

Reply:
Other views may differ, but whenever I've done this I've regretted it an ended up putting the test data in.

Reply:
Just that I ended up using the inputs for other things, waiting for the upstream tasks to run in CI etc. I won't fight you on it though :-D